src/main/java/examples/Examples.java

```diff
@@ -202,6 +202,23 @@ public void connecting05(Vertx vertx) {
     });
   }
 
+  public void connecting06(Vertx vertx) {
+
+    // Pool Options
+    PgPoolOptions options = new PgPoolOptions()
+      .setDomainSocket(true)
+      .setHost("/var/run/postgresql")
+      .setPort(5432)
```

Comment:
port is irrelevant

Reply:
Please take a look at this: https://www.postgresql.org/docs/9.6/static/runtime-config-connection.html.
 It's said in `unix_socket_directories` part that 
socket file will be named as `${PATH}/.s.PGSQL.${SERVER_PORT}` , so the port is useful here.